pyuvdata/uvcal/uvcal.py

```diff
@@ -505,6 +521,11 @@ def __init__(self):
             "Nsources", required=False, expected_type=int, description=desc
         )
 
+        desc = "Number of sources used."
+        self._Nphase = uvp.UVParameter(
+            "Nphase", required=False, expected_type=int, description=desc
+        )
+
```

Comment:
This description confuses me. I think Nphase is the number of phase centers NOT the number of sources in the sky model used for calibration.

Reply:
Yes, sorry -- this was a sloppy copy/paste after I realized that Nsources shouldn't be used like Nphase.

Reply:
Fixed in 766e32646768181b70ddd8d2e576d90ca57c254c.